docs/release-notes/cloud/cloud-2024-01-DD-gh-repository-ssh-keys-deprecation.md

```diff
@@ -0,0 +1,82 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+---
+
+# Deprecation of SSH keys for GitHub repositories January DD, 2024<!--TODO PLUTO-802 Update release date-->
+
+!!! info "This release note applies only to GitHub"
+
+For increased security, Codacy deprecated the usage of **repository SSH keys** for Git operations on GitHub in favor of **installation access tokens**.
+
+This change translates into important security improvements:
+
+-   It reduces the liability of the Codacy GitHub App, as it no longer requires [read and write repository permissions for Administration](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-administration).
```

Comment:
Can we reformulate this in the positive?

```suggestion
-   It improves the security of the Codacy GitHub App, as it no longer requires [read and write repository permissions for Administration](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-administration).
```

Reply:
I see your point... However, I would like to use here a word related to "liability", and I'm not finding a good positive expression to replace it.
"It improves the security" kind of repeats the stem sentence "This change translates into important **security improvements**"... 🤔 

I'll keep the current expression for now. 🙏